pkg/kyc/component.go

```diff
@@ -326,3 +328,22 @@ func (c *component) reportEvent(ctx context.Context, apiCall string, values ...s
 func ptr(value string) *string {
 	return &value
 }
+
+func (c *component) SendSmsCodeInSocialRealm(ctx context.Context, userID string) (string, error) {
+	return c.sendSmsCode(ctx, c.socialRealmName, userID)
+}
+
+func (c *component) sendSmsCode(ctx context.Context, realm string, userID string) (string, error) {
+	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
+
+	smsCodeKc, err := c.keycloakClient.SendSmsCode(accessToken, realm, userID)
```

Comment:
our keycloak API do not restrict the realm used to send the SMS ?
I would have expected that we need to use technical user to be able to send SMS code in a cross realm.